app/bibleview-js/src/components/TextEditor.vue

```diff
@@ -29,6 +29,7 @@
 import {inject, onMounted, onUnmounted, watch} from "@vue/runtime-core";
 import {ref} from "@vue/reactivity";
 import {useCommon} from "@/composables";
+import {bcv_parser} from "@/composables/refparser.js";
```

Comment:
Why not use english version straight from the package?

Reply:
Because I think the English version is essential while the other languages should be optional. I didn't want to import from node_modules since I assumed that that would mean all are included.

Reply:
if we import only english, only english will be in the js bundle